### DISCUSSION TITLE
Question about the evaluation

### DISCUSSION THREAD

**Discussion by luckyyangrun** [PaddlePaddle/PaddleSpatial#58] [BaiduKDDCup2022] [answered]
Dear organizer:
**What is the correct shape of prediction?**
![image](https://user-images.githubusercontent.com/67430592/168417323-d63215c5-712c-462b-a00b-6125d0e6f788.png)
Our code (under PyTorch framework) returns a  numpy.ndarray with shape [134, 288, 1] through forecast() interface, but we got the error :
**'operands could not be broadcast together with shapes (134,) (144,) '**

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Please refer to our baseline for test prediction https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru 

But sorry that we do not implement it in PyTorch, and we cannot answer it. 

Thanks a lot for your understanding. 

Regards

**Comment by ramber1836** [comment]
> Dear organizer: **What is the correct shape of prediction?** ![image](https://user-images.githubusercontent.com/67430592/168417323-d63215c5-712c-462b-a00b-6125d0e6f788.png) Our code (under PyTorch framework) returns a numpy.ndarray with shape [134, 288, 1] through forecast() interface, but we got the error : **'operands could not be broadcast together with shapes (134,) (144,) '**

I guess you can repack the returned numpy.ndarray with shape [134, 288, 1] to a collection of numpy.ndarrays where each one with the shape [288, 1]. Hope it works.

**Comment by ramber1836** [comment]
Correct, the shape of the prediction is [134, 288, 1].
Have you tested your code with the released test toy data?
